src/test/resources/keycloak/users.json

```diff
@@ -0,0 +1,62 @@
+{
+  "users": [
+    {
+      "username": "jdoe",
+      "firstName": "Jane",
```

Comment:
For consistency lets rename this to John

Reply:
That is a test file; it doesn't really matter.